lib/triggers/wikiBotTrigger.js

```diff
@@ -221,48 +221,48 @@ WikiBotTrigger.prototype._importGames = function(toId,steamId,query) {
 						var afterimport = function (title, revision, date) {
 							if(revision && date && title) {
 								that._sendMessageAfterDelay(toId,"Revision #" + revision + " completed at " + date.toString() + ". Data imported for "+result.appid+"/"+result.gamename);
-								that.winston.info(that.name+": Revision #" + revision + " completed at " + date.toString() + ". Data imported for "+result.appid+"/"+result.gamename);
+								that.winston.info(that.chatBot.name+"/"+that.name+": Revision #" + revision + " completed at " + date.toString() + ". Data imported for "+result.appid+"/"+result.gamename);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/944346689/issues/source?bid=1980036&fileBranchId=2848073#l224)